pandas_gbq/gbq.py

```diff
@@ -6,57 +6,19 @@
 import time
 import sys
 
+import google.auth
```

Comment:
no, you have to have a ``_try_imports`` to guard against these missing. The parent package (pandas) *only* checks that gbq itself can be imported, w/o a helpful message (hey you are missing a dependency).